examples/data-sources/nios_dtc_record_a/data-source.tf

```
@@ -0,0 +1,15 @@
+// Retrieve a specific DTC record A in a DTC server using filters 
+data "nios_dtc_record_a" "get_dtc_record_a_in_dtc_server_using_filters" {
+  filters = {
+    dtc_server = "example-server"
+    ipv4addr   = "2.21.2.3"
+  }
+}
+
+
+// Retrieve all DTC record A in a DTC server
```

```suggestion
// Retrieve all DTC A Records in a DTC server
```
Please check this for other examples too